src/main/resources/dictionaries/parse/license.txt

```diff
@@ -0,0 +1,59 @@
+# Dictionary to lookup a License enumeration by its URI or its acronym/title.
+# Non-CC licenses that GBIF considers equal to one of its three supported CC licenses can be added to this file.
+# To be explicit, widely used licenses GBIF does not support can be added to this file.
+# When adding new mappings, please separate them by CC version number.
+#
+# Supported CC 1.0 licenses
+http://creativecommons.org/publicdomain/zero/1.0	CC0_1_0
+CC0	CC0_1_0
+CC0 1.0	CC0_1_0
+cc-zero	CC0_1_0
```

Comment:
the parser is configured to allow digits, thereby just removing whitespace. 
To catch more cases without the need to an extensive dictionary file we could implement a better normalize method that case insensitively ignores all characters but a-z0-9? And maybe strips the http:// prefix or even the entire first part of the url http://creativecommons.org/licenses?